### PR TITLE
Creating a fetcher to fetch new icons

### PR DESCRIPTION
**Pull request by ganimtron-10**
Creating a fetcher function to fetch the new icons required for the `DrawPanel UI`.

Demo:

```python

from fury.data import fetch_viz_new_icons

fetch_viz_new_icons()

```

### CODE REVIEW

File: fury/data/fetcher.py [hunk 1]
```diff
@@ -24,6 +24,9 @@
 UW_RW_URL = \
     "https://digital.lib.washington.edu/researchworks/bitstream/handle/"
 
+NEW_ICONS_DATA_URL = \
+    "https://raw.githubusercontent.com/ganimtron-10/fury-data/master/icons/new_icons/"
```
Comment (ganimtron-10) [inline fury/data/fetcher.py:28]: Once this [PR](https://github.com/fury-gl/fury-data/pull/9) gets merged I will update the link with the official fury-data repo.

**Comment by m-agour** [comment]
Hello @ganimtron-10,
Thanks for including the demo. I tested the fetcher and works like a charm.

**Comment by ganimtron-10** [comment]
Hello @skoudoro ,
I have made the required changes.
PTAL.
Thanks!

**Review by skoudoro — APPROVED** [review]
Thanks!

**Pull request merged**